cpp/src/sampling/random_walks_impl.cuh

```diff
@@ -70,6 +72,18 @@ struct sample_edges_op_t {
   }
 };
 
+template <typename vertex_t, typename bias_t>
+struct e_bias_op_t {
+
+  raft::device_span<bias_t const> vertex_weight_sum{};
+
+  __device__ bias_t
+  operator()(vertex_t src, vertex_t, thrust::nullopt_t, thrust::nullopt_t, bias_t weight) const
+  {
+    return weight / vertex_weight_sum[src];
+  }
+};
```

Comment:
You may have another e_bias_op_t in the future for node2vec random walk. You may name this considering this future updates (e.g. biased_random_walk_e_bias_op_t).